### ISSUE TITLE
Change Gulp Tasks to be Asynchronous 

### ISSUE THREAD

**Issue by buschschwick** [factor1/factor1_blankwp#10]
A bug was found that would cause an indefinite loop based on the gulp task `watch` being fired too soon when files were compiling. 

The current fix was to remove `watch` from the `default` task. 

In 2.1.0 the plan is to set up tasks asynchronously too allow for correct orders.

**Comment by buschschwick** [comment]
minify-css is now waiting on sass before it can process. This was added with commit https://github.com/factor1/factor1_blankwp/commit/f7d6463d909f40918cbb569003a28b536fdfed0d on the branch 2.1.0

Note: it has not been tested yet and further testing is needed before it can be approved and merged.

**Comment by buschschwick** [comment]
Tested, and merged into master branch.
